L1Trigger/Phase2L1ParticleFlow/interface/ParametricResolution.h

```diff
@@ -0,0 +1,67 @@
+#ifndef L1Trigger_Phase2L1ParticleFlow_ParametricResolution_h
+#define L1Trigger_Phase2L1ParticleFlow_ParametricResolution_h
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include <vector>
+#include <cmath>
+
+namespace l1tpf {
+  class ParametricResolution {
+  public:
+    ParametricResolution() {}
+    ParametricResolution(const edm::ParameterSet &cpset) {
+      std::vector<double> etaBins = cpset.getParameter<std::vector<double>>("etaBins");
```

Comment:
ok

Reply:
is there already some predefined snipped to read `cms.vdouble` into a  `vector<float>` ? 
otherwise I can add a converter or use some stl & lambda's but it may make the code less readable than what it is now (probably without any real gain in performance, since this happens only at initialization time)

Reply:
I suspect such a converter doesn't currently exist. A simple function like this could be used:
```cpp
std::vector<float> asVecFloat(const std::vector<double>& vd){
  return std::vector<float>(vd.begin(),vd.end());
}
```

Reply:
ok done